Signal/src/ViewControllers/OWSConversationSettingsViewController.m

```diff
@@ -827,10 +827,7 @@ - (void)showUpdateGroupView:(UpdateGroupMode)mode
     updateGroupViewController.conversationSettingsViewDelegate = self.conversationSettingsViewDelegate;
     updateGroupViewController.thread = (TSGroupThread *)self.thread;
     updateGroupViewController.mode = mode;
-
-    UINavigationController *navigationController =
-        [[UINavigationController alloc] initWithRootViewController:updateGroupViewController];
-    [self presentViewController:navigationController animated:YES completion:nil];
+    [self.navigationController pushViewController:updateGroupViewController animated:YES];
```

Comment:
I don't know why we were presenting the "edit group" view as a new modal instead of pushing it.